### DISCUSSION TITLE
Slider not appearing

### DISCUSSION THREAD

**Discussion by hickorysb** [fomantic/Fomantic-UI#1288] [General] [answered]
```html
<script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
<script src="semantic/dist/semantic.min.js"></script>
<link rel="stylesheet" type="text/css" href="semantic/dist/semantic.min.css">
```
```html
<body>
    <div class="ui slider"></div>
</body>
```
```js
$('.ui.slider').slider();
```

The slider isn't appearing. I don't know why. Any help is appreciated.
Fomantic-UI v2.8.3

**Comment by lubber-de** [comment]
Your code is fine. It seems you have put semantic.min.css and semantic.min.js to a different location, so the browser cannot find them by the relative path in your code
See here https://jsfiddle.net/bhs0wz3f/

Try to use jsdelivr cdn
```html
<script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.3/dist/semantic.min.js"></script>
<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.3/dist/semantic.min.css">
```

**Comment by hickorysb** [comment]
The CDN did not help. I am using Electron if that helps. I have a table and button working fine with Fomantic-UI right now.

EDIT:
I tried opening it in my browser and it still doesn't work, I don't think it's an issue with Electron. I created two test files.
index,html
```html
<!DOCTYPE html>
<html>
    <head>
        <script src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>
        <script src="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.3/dist/semantic.min.js"></script>
        <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/fomantic-ui@2.8.3/dist/semantic.min.css">
        <script src="js/index.js"></script>
    </head>
    <body>
        <div class="ui slider"></div>
    </body>
</html>
```
index.js
```js
$('.ui.slider').slider();
```

The browser finds and loads everything, but does not load the slider.

**Comment by exoego** [comment]
Could you ensure that `js/index.js` is loaded?
You can confirm it in the network tab of browsers' devtools.

Also, sharing debug console log of
`$('.ui.slider').slider({ debug: true });`
might help.

**Comment by hickorysb** [comment]
No output is logged, index.js is definitely loaded.
![image](https://user-images.githubusercontent.com/44131234/73143672-6e90e000-4049-11ea-9d3a-0774f8dae679.png)

**Comment by hickorysb** [comment, marked as answer]
I fixed it. It was being run before the page was initialized. I wasn't calling it in an onload function.

**Comment by exoego** [comment]
Ah, that is a common mistake 😉 

Closing as resolved.
